tests/Zend/Locale/FormatTest.php

```diff
@@ -35,6 +35,12 @@
  */
 class Zend_Locale_FormatTest extends PHPUnit_Framework_TestCase
 {
+    /**
+     * Constant for Non-breaking space UTF-8 encoded value.
+     * https://en.wikipedia.org/wiki/Non-breaking_space
+     */
+    const NBSP = " ";
```

Comment:
Perhaps this would be safer and more explicit? 🤔 (not sure if it will work)
```suggestion
    const NBSP = "\u{00A0}";
```

Reply:
unicode literals require php 7.0

```
➔ php53 -r 'var_dump("\u{00A0}");'
string(8) "\u{00A0}"
➔ php56 -r 'var_dump("\u{00A0}");'
string(8) "\u{00A0}"
➔ php70 -r 'var_dump("\u{00A0}");'
Command line code:1:
string(2) " "
```

Reply:
Safe, was to introduce a constant :)

Reply:
Ok... perhaps `"\xE2\x80\x8A"`? This should be compatible with php5...
I am not a fan of using invisible/indistinguishable special unicode characters in places where they can get lost over time in an accidental change of file encoding 🙊

Reply:
hmm, wrong byte order?

```
➔ php70 -r 'var_dump([$a="\u{00A0}", $b="\xE2\x80\x8A", "\u{00A0}" === "\xE2\x80\x8A"]);'
Command line code:1:
array(3) {
  [0] =>
  string(2) " "
  [1] =>
  string(3) " "
  [2] =>
  bool(false)
}
```

I'll try to find the byte value on my own too, maybe you messed up :)

Reply:
```
➔ php70 -r 'var_dump("\u{00A0}" === "\xC2\xA0");'
Command line code:1:
bool(true)
```

Reply:
I just shamelessly copied it from https://stackoverflow.com/a/48554122/3729316 without checking 😅